### ISSUE TITLE
[Adaptive cards]: Screen reader announces extra information as 'App bar' with all the header controls.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9236]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0
Screen Reader: Narrator/NVDA

**Repro steps:**

1. Open the Adaptive Card Visualizer application.
2. Turn on the Narrator through Ctrl+ windows+ enter key.
3. Navigate to the header control through tab key.
4. Observe the issue.

**Actual Result:**
On all the header controls such as on 'New card', 'Save file', 'Open file' and on 'Host config', screen reader announces extra information as app bar.

**Observation:**
Same issue repro with NVDA and JAWS.

**Expected Result:**
Screen reader should not announce extra information as 'App bar' with all the header controls.

**User Impact:**
Person with cognitive disability will get impacted if extra information is announced by the screen reader and it appears to be too verbose to the user which might confused the user or user will not be able to interact with the content properly.

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/f7370c7b-a7ff-4987-986f-000597209d61)

https://github.com/user-attachments/assets/6412b304-f576-4af8-a368-d7bcf92abc05

### Card JSON

```JSON
N/A
```
